output/style.css

```diff
@@ -93,15 +100,14 @@ nav > i {
 
 .background {
   width: 100%;
-  height: 587px;
   background-image: url("Microverse-Student-Project-1/header-shapes\ mobile.png");
   background-size: cover;
 }
 
 .text-container {
   display: flex;
   flex-direction: column;
```

Comment:
- [OPTIONAL] At the breakpoints 767px,  background image at your headline section does not extend to cover the whole width of that section. 
- Also, the project information of each projects is not also responsive at that breakpoints. They seem to be fixed to a particular width at that breakpoints. Kindly look into them and fix them.

[your-page-blank-space.webm](https://user-images.githubusercontent.com/26597432/215273688-45687bd6-352e-48dd-b642-866faecf58c6.webm)